packages/block-library/src/gallery/editor.scss

```diff
@@ -12,6 +12,11 @@ ul.wp-block-gallery {
 	}
 }
 
+// need to override default editor ul styles
+.blocks-gallery-grid.blocks-gallery-grid {
```

Comment:
Why twice?

Reply:
I left the comment above in an attempt to clarify, but perhaps it should be more explicit. I needed to override styles that are set for all `ul` in the editor, here:
<img width="183" alt="Screen Shot 2019-08-21 at 11 24 56 am" src="https://user-images.githubusercontent.com/8096000/63395243-7303fd80-c406-11e9-93bb-1f432cf8186e.png">
I like this technique of duplicating the selector to increase specificity because it makes it clear that there is no other reason for use of more than one selector. Chaining two different selectors or using a parent selector might be necessary to target a particular element, but chaining the same selector twice can only be used for specificity increase, so if we revisit this later and find that the style being overridden is gone, we'll know it's safe to remove the extra selector 🙂

Reply:
But when does `.blocks-gallery-grid.blocks-gallery-grid` occur? This means an element will have the class twice?

Reply:
No, the element only needs to have the class once. Chaining the same class two or more times together will only increase specificity for that rule. It'll still apply to any element that has that class on it.